gradio/components/checkboxgroup.py

```diff
@@ -109,8 +113,8 @@ def get_config(self):
 
     def example_inputs(self) -> dict[str, Any]:
         return {
-            "raw": self.choices[0] if self.choices else None,
-            "serialized": self.choices[0] if self.choices else None,
+            "raw": self.choices[0][1] if self.choices else None,
```

Comment:
this is actually incorrect and should be 
`[self.choices[0][1]] if self.choices else None` for both raw and serialized (its also incorrect in main). Should be this way, similar to Dropdown multiselect, otherwise the code snippets in api docs don't work as expected.

Reply:
Yeah you're right